HA/xcatha.py

```diff
@@ -748,7 +788,7 @@ def xcatha_setup_mn(self, args):
                 self.check_shared_data_db_type(args.dbtype,args.path)
             if self.configure_vip(args.virtual_ip,args.nic,args.netmask):
                 return 1
-            self.save_original_host_and_ip()
+            #self.save_original_host_and_ip()
```

Comment:
Why do you comment this line？ 
Before we change hostname for MN during setup process, we should save original hostname like in some place /tmp/ha_mn or /etc/ha_mn. When we want to restore original  hostname, we can find them.  User cannot identify which hostname is original one or virtual one. so original physical hostname should be kept somewhere.

Reply:
Yes, my mistake. I commented this out for UT, and forgot to undo.
Fixed now.